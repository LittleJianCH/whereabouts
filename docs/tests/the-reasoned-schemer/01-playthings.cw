Succeed []

Fail []
-------- {
  unify 1 = 2
}

query (q) {
  Fail []
}

failure {
  Fail []
}

success {
  Succeed []
}

query (q) {
  unify "pea" = "pod"
}

query (q) {
  unify q = "pea"
}

query (q) {
  unify "pea" = q
}
