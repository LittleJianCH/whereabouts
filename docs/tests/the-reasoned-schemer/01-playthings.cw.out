{ "success": false, "count": 0, "solutions": [] }
{ "success": false, "count": 0, "solutions": [] }
{ "success": true, "count": 1, "solutions": [{ "q": "pea" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "pea" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "#unknown" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "#unknown" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "#unknown" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "?x" }] }
{ "success": true, "count": 1, "solutions": [{ "q": ["?x"] }] }
{ "success": true, "count": 1, "solutions": [{ "q": "#unknown" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "?x" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "pod" }] }
{ "success": true, "count": 1, "solutions": [{ "q": ["?x", "?x"] }] }
{ "success": true, "count": 1, "solutions": [{ "q": ["?x", "?y"] }] }
{ "success": true, "count": 1, "solutions": [{ "s": ["?t", "?u"] }] }
{ "success": true, "count": 1, "solutions": [{ "q": ["?x", "?y", "?x"] }] }
{ "success": false, "count": 0, "solutions": [] }
{ "success": false, "count": 0, "solutions": [] }
{ "success": true, "count": 1, "solutions": [{ "q": "#unknown" }] }
{ "success": true, "count": 1, "solutions": [{ "q": "corn" }] }
{ "success": false, "count": 0, "solutions": [] }
{ "success": false, "count": 0, "solutions": [] }
{ "success": false, "count": 0, "solutions": [] }
{ "success": true, "count": 1, "solutions": [{ "q": "corn" }] }
{ "success": true, "count": 2, "solutions": [{ "q": "olive" }, { "q": "oil" }] }
